sdk/sdk-server.py

```diff
@@ -80,6 +80,42 @@ def get_player_career(player_id):
 
     return player_career_stats_all
 
+
+@app.route('/player/<int:player_id>/vs/<int:vs_player_id>', methods=['GET'])
+def get_player_vs_player(player_id, vs_player_id):
+    """
+    Retrieves the head-to-head statistics between two players.
+
+    Parameters:
+    player_id (int): The ID of the first player.
+    vs_player_id (int): The ID of the second player.
+
+    Returns:
+    dict: A dictionary containing the head-to-head statistics between the two players.
+
+    """
+
+    season = request.args.get('season', '2023-24')
```

Comment:
add all the params from sdk… make it available to the other end.

Reply:
done. please review.